### DISCUSSION TITLE
when I use climate_indices in python, I met a problem, please help me

### DISCUSSION THREAD

**Discussion by LLLLincoln** [monocongo/climate_indices#471] [Q&A]
Hello everyone,
   when I use climate_indices in python, I met a problem, please help me! Detail information list below:
![image](https://user-images.githubusercontent.com/45530552/179416660-85f8ca7d-93f8-4d51-b72e-32c83ce1333f.png)
Thanks in advance

**Comment by monocongo** [comment]
Sorry, this is related to NCO which is a dependency that should be removed from this package, it rarely works well for Windows users.

The best way to avoid this error is to re-orient the dimensions of your datasets to (lat, lon, time), which is what the code is attempting to do via NCO (pynco wrapper actually), but NCO is developed for Linux and we frequently see issues from users who use Windows, just like this. It may be that you don't have NCO installed, if so then do that and try again, that may be all you need to do. Otherwise, you can re-orient the datasets as described above (but you may need NCO for that anyway, so try the NCO installation first).

pynco/NCO installation details are here: https://pynco.readthedocs.io/en/latest/

**Reply by LLLLincoln** [reply]
Thanks, I try it in linux, but the output of SPEI is NAN, 
![image](https://user-images.githubusercontent.com/45530552/179568747-a57af611-5fc2-4ba0-9d88-2f23ef829ea9.png)
but my pet and precipitation is not NAN

**Comment by monocongo** [comment]
If you can post your PET and precipitation files then I may be able to use those to find the problem. No promises, but we can try...

**Comment by LLLLincoln** [comment]
The PET and precipitation data
https://drive.google.com/file/d/1wum-ucP2qGiq7oWBWYMLERYDXtmcwpan/view?usp=sharing
